src/sonic-bgpcfgd/tests/test_static_rt.py

```diff
@@ -71,6 +71,45 @@ def test_set():
         ]
     )
 
+def test_set_nhportchannel():
+    mgr = constructor()
+    set_del_test(
+        mgr,
+        "SET",
+        ("10.1.0.0/24", {
+            "nexthop": "PortChannel0001",
+        }),
+        True,
+        [
+            "ip route 10.1.0.0/24 PortChannel0001",
+            "router bgp 65100",
+            " address-family ipv4",
+            "  redistribute static",
+            " address-family ipv6",
+            "  redistribute static"
+        ]
+    )
```

Comment:
Could you please add in this test case to confirm the portchannel routes can be properly removed?